Map Loader/Program.fs

```diff
@@ -253,15 +340,14 @@ let initialiseCache () =
     )
   users <- users |> mergeMaps alreadyParsedUsers |> mergeMaps _m2
   File.WriteAllText("users.json", users |> JsonConvert.SerializeObject)
-  Console.CursorTop <- Console.CursorTop - 1
-  colorprintfn "$darkgray[Updated users ]"
-
-let version = "a1"
+  //Console.CursorTop <- Console.CursorTop - 1
+  //colorprintfn "$darkgray[Updated users ]"
+  Console.Title <- sprintf "Map Loader %s" version
 
 let () =
   Console.Title <- sprintf "Map Loader %s" version
   colorprintfn "$yellow[Map Loader %s by Skylark#6969.\nPlease see https://skylarkx.uk/maploader for help.\nThere are %i maps available]" version maps.Length
-  let latestVersion = (new WebClient()).DownloadString("https://raw.githubusercontent.com/teenangst/maploader/main/version.txt").Split('\n')
+  let latestVersion = (new WebClient()).DownloadString(sprintf "https://raw.githubusercontent.com/teenangst/maploader/main/version.txt?%i" unixtime).Split('\n')
```

Comment:
consider moving to `HttpClient`
see also: https://stackoverflow.com/questions/20530152/deciding-between-httpclient-and-webclient